### DISCUSSION TITLE
[SOLVED] Nuxt 3 Very slow with Vite

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [nuxt/nuxt#16853] [Questions]
Why is Nuxt 3 very slow if it's using Vite? 

Maybe Nitro's fault? 🤔

I'm considering using pure Vue 3 Vite projects for my next YouTube tutorials because this really slows down my workflow. The only reason I switched to Nuxt 3 was so I don't have to create Vue-Router router and all the routes manually all the time (and layouts) and also because of the auto-imports. Not really needing SSR right now I think.

Hope this can be solved though. Wishing the team health and success.

**Comment by danielroe** [comment]
would you provide an example project? 🙏

**Reply by angelhdzmultimedia** [reply]
Sorry it took me too long to reply, I got a strange issue creating a new project that I didn't have before, all the times I made a Nuxt 3 project. Using `3.0.0-rc.10` now, `3.0.0-rc.9` yesterday.

Now I'm wondering if the issue is my Node version **18.1.0** conflicting... 

![image](https://user-images.githubusercontent.com/10191085/190523768-c4dfd5b7-2e8c-4486-b9de-532713d16cf9.png)

**Reply by angelhdzmultimedia** [reply]
Was recording my screen to show the "slowness", but as I got this error, I stopped the recording and now I'm grabbing my hairs why is this error happening. I'm doing all the same steps I've done before (and I have Nuxt 3 videos on YouTube working).

First I run `npx nuxi init` to create the project. Then I run `npm i `to install deps. Finally run `npm run dev`.

**Reply by angelhdzmultimedia** [reply]
![image](https://user-images.githubusercontent.com/10191085/190524827-8fd1f66b-cfc9-4c09-bf4a-076f10a22a25.png)

**Reply by angelhdzmultimedia** [reply]
Going to try again tomorrow. And if I manage to create a working project, I'll post the link to the repo, and also the video to show the time it takes to serve the app and load the initial content in the browser.. Thanks for your time!

**Comment by angelhdzmultimedia** [comment]


**Reply by danielroe** [reply]
I'm sorry to hear that. But rest assured, you're welcome here 😊

**Reply by pi0** [reply]
@andershagbard I'm sorry to feel this way. Your comments in #7573 were marked as spam because they were not relevant and helping in resolving the issue.

Please check out the [contribution guide](https://v3.nuxtjs.org/community/contribution) about what is expected when you are contributing to an open source project.

**Reply by andershagbard** [reply]
@pi0 Think you meant to tag @angelhdzmultimedia 🤔
